### ISSUE TITLE
Using objects/closures in the config file

### ISSUE THREAD

**Issue by ye7iaserag** [Astrotomic/laravel-monolog-config#7]
The config file needs to be updated to not include any objects
use only class names
To repeat try the Laravel config:cache command, the generated cached configuration will always throw a fatal error
Please update the bundle
Thanks

**Comment by ye7iaserag** [comment]
The exact lines are 
new LineFormatter()
new Swift_Message('Laravel Log')

only class names should be used in configurations, example:
LineFormatter::class
Swift_Message::class

**Comment by Gummibeer** [comment]
The problem ist that some of these classes need parameters on construct.
Do you have an idea how to solve it?

**Comment by Gummibeer** [comment]
I could think about a solution like:
```
'message' => [
    'class' => Swift_Message::class,
    'args' => ['Laravel Log'],
]
```
This could be used by http://php.net/manual/de/reflectionclass.newinstanceargs.php to get a new instance of the given class including parameters and it is serializable.

**Comment by ye7iaserag** [comment]
Yes that's exactly what was recommended by laravel's config files guidelines

**Comment by ye7iaserag** [comment]
Deserializing big objects from the config files is costly process while bootstrapping the framework, that's why they went against supporting closures in config files

**Comment by Gummibeer** [comment]
Ok, I will Update it tomorrow and give or a test.

**Comment by Gummibeer** [comment]
@ye7iaserag it's now in `dev-master` - implemented with https://github.com/Astrotomic/laravel-monolog-config/commit/e69fcbc35a7ebd46efd3d5aff80023a6ca6c170b
Would be nice if you can give it a try, before I release it as `v2.0` cause it's a breaking change. 😃

**Comment by ye7iaserag** [comment]
Will test is today and update you, but from what i can see that was exactly what was needed
Good job :)

**Comment by Gummibeer** [comment]
@ye7iaserag any update?
